src/NServiceBus.Core/DataBus/DataBusProperty.cs

```diff
@@ -12,6 +12,13 @@
     public class DataBusProperty<T> : IDataBusProperty, ISerializable where T : class
     {
         /// <summary>
+        /// initializes a <see cref="DataBusProperty{T}" /> with no value set.
+        /// </summary>
+        public DataBusProperty() : this(null)
```

Comment:
@SimonCropp does the ctor actually need to be public? (I see no issue with it being public, it's mostly curiosity)